langchain/vectorstores/qdrant.py

```diff
@@ -78,7 +79,7 @@ def add_texts(
         """
         from qdrant_client.http import models as rest
 
-        ids = [uuid.uuid4().hex for _ in texts]
+        ids = [md5(_.encode('utf-8')).hexdigest() for _ in texts]
```

Comment:
```suggestion
        ids = [md5(txt.encode('utf-8')).hexdigest() for txt in texts]
```

Reply:
fixed, and local `make lint` passes. thank you